extensions/cocostudio/timeline/CCBoneNode.js

```diff
@@ -308,9 +313,8 @@ ccs.BoneNode = (function () {
         },
 
         setVisible: function (visible) {
-            if (this._visible == visible || !this._rootSkeleton)
+            if (this._visible == visible)
                 return;
```

Comment:
need dirty the bones' sequence SkeletonNode::_subOrderedAllBones for draw skins as zorder